pkgs/by-name/im/immich/package.nix

```diff
@@ -166,6 +167,13 @@ buildNpmPackage' {
     # see https://github.com/immich-app/immich/issues/13971
     substituteInPlace src/services/backup.service.ts \
       --replace-fail '`/usr/lib/postgresql/''${databaseMajorVersion}/bin/pg_dumpall`' '`pg_dump`'
+
+    # some part of the build wants to use un-prefixed binaries. let them.
+    mkdir -p $TMP/bin
```

Comment:
I've seen `$TMPDIR` before. What's the difference and where is `$TMP` set?

Reply:
TMP, TEMP, TMPDIR, and TEMPDIR are all synonyms, set by nix itself! https://github.com/NixOS/nix/blob/2c1447b7f2ed523631471b9a232b462d5e7fb075/src/libstore/unix/build/local-derivation-goal.cc#L1169